src/base/bittorrent/session.h

```diff
@@ -503,6 +504,9 @@ namespace BitTorrent
 
         void findIncompleteFiles(const TorrentInfo &torrentInfo, const QString &savePath) const;
 
+        bool isReannounceWhenAddressChanged() const;
```

Comment:
Move them below `setMaxConcurrentHTTPAnnounces()` declaration. This applies to .cpp file too.